src/watertap_contrib/seto/solar_models/surrogate/data/pysam_run.py

```diff
@@ -0,0 +1,326 @@
+import json
+from os.path import join, dirname
+from math import floor, ceil, isnan
+import numpy as np
+import pandas as pd
+import time
+import multiprocessing
+from itertools import product
+import matplotlib.pyplot as plt
+import PySAM.TroughPhysicalProcessHeat as iph
+import PySAM.IphToLcoefcr as iph_to_lcoefcr
+import PySAM.Lcoefcr as lcoefcr
+
+
+def read_module_datafile(file_name):
+    with open(file_name, "r") as file:
+        data = json.load(file)
+    return data
+
+
+def load_config(modules, file_names=None, module_datas=None):
+    for i in range(len(modules)):
+        if file_names is not None:
+            data = read_module_datafile(file_names[i])
+        elif module_datas is not None:
+            data = module_datas[i]
```

Comment:
```suggestion
            data = module_data[i]
```
This assumes that `modules` is the same length as `file_names` or `module_data`. Will that always be the case? 

Also, I am guessing that these can be a single string or a list of strings. Should we enforce types? Also, could you add some doc strings to explain args a bit more?